fast64_internal/oot/tools/quick_import.py

```diff
@@ -40,7 +40,7 @@ def quick_import_exec(context: bpy.types.Context, sym_name: str):
 
     all_found_defs: dict[Path, list[tuple[str, str]]] = dict()
 
-    for dirpath, dirnames, filenames in os.walk(assets_objects_dir_p):
+    for dirpath, _, filenames in os.walk(assets_objects_dir_p):
```

Comment:
x2
and this is also a super common construct
```suggestion
    for dirpath, dirnames, filenames in os.walk(assets_objects_dir_p):
```